src/logger/transports/console.mjs

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+import { prettifyLevel } from '../utils/colors.mjs';
+import { prettifyTimestamp } from '../utils/time.mjs';
+
+/**
+ * Logs a formatted message to stdout for human-friendly CLI output.
+ *
+ * @param {import('../types').TransportContext} context
+ * @returns {void}
+ */
+export const console = ({
```

Comment:
I wonder the compatibility of this for indeed work done by generators 🤔 

Either generators will need to run their own logger instances individually if they want to log OR we only log high-level progress between each generator.

Although I'd wish the progress would be more granular:

- for example instead of progress bar prgressing when each generator finishes, having it progress when each file gets updated is chef-kiss imo.

Reply:
We could also send the logs to the parent, and have it to the processing, but that's more complex?